readme.md

```diff
@@ -154,6 +154,7 @@ Creates a new client instance.
 	* `password`: The password for the BrowserStack account.
 	* `version` (optional; default: `4`): Which version of the BrowserStack API to use.
 	* `server` (optional; default: `{ host: "api.browserstack.com", port: 80 }`): An object containing `host` and `port` to connect to a different BrowserStack API compatible service.
+	* `proxy` (optional; default: `null`): Proxy server supporting HTTPS to be used for connecting to BrowserStack (or `settings.server`). e.g. `"http://proxy.example.com:1234"`
```

Comment:
Proxies listening on non-SSL ports can proxy HTTPS connections too, but sure, I'll change the sample link to HTTPS.